api/helpers/gameStates/deal-cards.js

```diff
@@ -0,0 +1,73 @@
+const GameStatus = require('../../../utils/GameStatus.json');
+const DeckIds = require('../../../utils/DeckIds.json');
+const MoveType = require('../../../utils/MoveType.json');
+const GamePhase = require('../../../utils/GamePhase.json');
+
+module.exports = {
+  friendlyName: 'Deal Cards',
+
+  description: 'Creates initial GameStateRow for specified game, with cards dealt ot both players (5 for p0 and 6 for p1). Errors if Game has previous game or incorrec status.',
+
+  inputs: {
+    game: {
+      type: 'ref',
+      description: 'Game record to be initialized with first GameState',
+      required: true,
+    },
+  },
+
+  fn: async ({ game }, exits) => {
+    
+    try {
+      if (game.status !== GameStatus.CREATED) {
+        return exits.error({ message: 'Game has already started' });
+      }
+
+      const previousGameStates = await GameStateRow.find({ gameId: game.id});
+      if (previousGameStates?.length) {
+        return exits.error({ message: 'Cards are already dealt' });
+      }
+
+      const deck = _.shuffle(
+        DeckIds.map(sails.helpers.gameStates.convertCardIdToCard)
```

Comment:
Helper name changed. Also need the .map() callback to pass the arg because sails helpers input a single object arg
```suggestion
        DeckIds.map((str) => sails.helpers.gameStates.convertStrToCard({ str }))
```